platformics/codegen/templates/api/types/class_name.py.j2

```diff
@@ -507,7 +507,7 @@ async def create_{{ cls.snake_name }}(
     cerbos_client: CerbosClient = Depends(get_cerbos_client),
     principal: Principal = Depends(require_auth_principal),
     is_system_user: bool = Depends(is_system_user),
-) -> db.Entity:
+) -> db.{{ cls.name }}:
```

Comment:
These return the current type, there's no reason to tell the type checker that we return the base type. I think. 😅